internal/internal_workflow_testsuite_test.go

```diff
@@ -1260,8 +1260,8 @@ func (s *WorkflowTestSuiteUnitTest) Test_QueryWorkflow() {
 	stateWaitSignal, stateWaitActivity, stateDone := "wait for signal", "wait for activity", "done"
 	workflowFn := func(ctx Context) error {
 		var state string
-		err := SetQueryHandler(ctx, queryType, func() (string, error) {
-			return state, nil
+		err := SetQueryHandler(ctx, queryType, func(queryInput string) (string, error) {
```

Comment:
May be test more than one arg just to be sure?

Reply:
i have tested 2 args as well